bundles/org.openhab.binding.lutron/README.md

```diff
@@ -368,6 +369,33 @@ Thing configuration file example:
 Thing shade libraryshade [ integrationId=33]
 ```
 
+### Blinds [**Beta**]
+
+Each Lutron Sivoia QS Venetian Blind or Horizontal Sheer Blind is controlled by a **blind** thing.
+Besides `integrationId`, it requires that the parameter `type` be set to either "Venetian" for venetian blinds or "Sheer" for horizontal sheer blinds.
+There is no default.
+If discovery is used, the `type` parameter will automatically be correctly set when the **blind** thing is created.
+
+Two channels, *blindliftlevel* and *blindtiltlevel*, with item type Rollershutter and category Rollershutter will be created for each **blind** thing.
+They control the up/down motion and the slat tilt motions of the blinds, respectively.
+Each channel accepts Percent, Up, Down, Stop and Refresh commands.
+Sending a Percent command will cause the blind to immediately move so as to be open the specified percentage.
+You can also read the current setting from each channel.
+It is specified as a percentage, where 0% = closed and 100% = fully open. Movement delays are not currently supported.
+
+**Note:** While a blind is moving to a specific level because of a Percent command, the Lutron system will report the target position for the blind rather than the actual current position.
+While a blind is moving because of an Up or Down command, it will report the previous level until it stops moving.
+
+**Beta Notice:** Support for Sivoia QS blinds is new and has been through very limited testing.
+Please comment on your use of it in the openHAB [community forum](https://community.openhab.org/).
```

Comment:
Please remove the references about the forum and GitHub issues. That is covered on other places. This binding documentation will be part of the overall documentation and binding documentation should not contain all kind of information to general places, unless they are specific for that binding.

Reply:
Ok. That makes sense. I'll change it to "Experimental", then remove the "Experimental" once I have confirmation that position updates are working properly on both types of blinds.

This has been an ongoing problem with this binding. Lutron had several systems, and each system has a large number of devices that can attach to it, and many devices have several different models or configuration variations. So I obviously can't own them all (as much as I might like to). But I want to move toward full device coverage to attract more users. When I add support for devices I don't own, I rely on the API docs, dummy devices configured on my own system, and various methods of simulation. But eventually, someone has to test it on each device to confirm that it really works in all cases. So I usually build a test version of the binding, put it on GitHub, and post messages in the forum begging for volunteers to test it. Sometimes this work, but sometimes it doesn't. Often it seems like people will only really start testing out new features once they make it in to a milestone release, but that means they need to be merged in, which presents sort of a chicken-and-egg problem, since I don't want to merge in code that I don't feel has been adequately tested.  So in this case, this was the compromise I came up with - create a PR for it, but document it as still needing further testing.

Anyway, sorry this is probably more venting than explanation!

Reply:
I can relate to your problem. I have the same with the TP-Link binding. I actually have only 1 device.
To handle this issue I've added unit tests and separated the device specific code as much as possible from openHAB logic to make it easier to test. With the new build system its easier to add unit tests.
And I've added a beta version to the eclipse iot marketplace. Although I don't know how long the marketplace will be available, give the move of core back to openHAB.

Reply:
Yeah, trying to add some unit tests is definitely on my "to do" list. Unfortunately, it's a pretty long list at the moment.
Are there any plans yet for an eclipse marketplace replacement?